rkt/run.go

```diff
@@ -166,13 +168,19 @@ func runRun(args []string) (exit int) {
 
 	pcfg := stage0.PrepareConfig{
 		CommonConfig: cfg,
-		Apps:         &rktApps,
-		Volumes:      []types.Volume(flagVolumes),
-		Ports:        []types.ExposedPort(flagPorts),
-		InheritEnv:   flagInheritEnv,
-		ExplicitEnv:  flagExplicitEnv.Strings(),
 		UseOverlay:   !flagNoOverlay && common.SupportsOverlay(),
 	}
+
+	if len(flagPodManifest) > 0 {
+		pcfg.PodManifest = flagPodManifest
+	} else {
+		pcfg.Volumes = []types.Volume(flagVolumes)
```

Comment:
I think we should just error out if any of these flags are set in addition to flagPodManifest..